app/code/Magento/NewsletterGraphQl/Model/Resolver/SubscribeEmailToNewsletter.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\NewsletterGraphQl\Model\Resolver;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAlreadyExistsException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\Validator\EmailAddress as EmailValidator;
+use Magento\Framework\App\Config\ScopeConfigInterface;
```

Comment:
Please, sort imports alphabetically (there's "Optimize imports" feature in PHPStorm, I recommend to enable the automated sorting by default)

Reply:
**done**

I've missed that...